indico/web/client/styles/modules/event_display/_common.scss

```diff
@@ -130,3 +130,46 @@
     }
   }
 }
+
+.reference-list {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  display: block;
+
+  li {
+    display: inline-block;
+    margin: 0.2em 0.2em;
+    border-radius: 2px;
+
+    .type-name {
+      color: $light-black;
+    }
+  }
+}
+
+.reference-list-meeting {
+  li {
+    &::before {
+      color: $yellow;
+    }
+
+    padding: 0.2em 0.5em;
+    background-color: lighten($light-yellow, 5%);
+
+    &:first-child {
+      margin-left: 0;
+    }
+  }
+}
+
+.reference-list-conference {
+  li {
+    &::before {
+      content: '';
+    }
```

Comment:
With my suggestion on Mattermost I meant to just avoid the yellow color, not removing the icon altogether ;)

Reply:
Like this?

![Screenshot from 2021-06-04 14-48-36](https://user-images.githubusercontent.com/8739637/120803598-01963000-c544-11eb-9b59-94810d2761e7.png)

Reply:
yeah